HISTORY.rst

```diff
@@ -3,14 +3,15 @@
 History
 -------
 
-0.7.0 (unreleased)
+1.0.0 (unreleased)
```

Comment:
``0.7.0`` is not released so we don't need to bump another version since its already backwards incompatible.

Reply:
If we're doing semver, then this is a backwards incompatible API change which requires a major version bump.